lib/instances/LocalizationTable.lua

```diff
@@ -0,0 +1,34 @@
+local BaseInstance = import("./BaseInstance")
+local HttpService = import("./HttpService")
+local InstanceProperty = import("../InstanceProperty")
+local LocalizationTable = BaseInstance:extend("LocalizationTable")
+
+local httpService = HttpService:new()
```

Comment:
Hmm, this is probably not a good pattern, since instantiating multiple copies of the same service is dubious.

I feel like we should break out the wrapper around `dkjson` into its own module and load that instead of loading `HttpService` anywhere that touches JSON? Maybe that belongs in a separate PR.